src/hooks/__tests__/customFetch.test.js

```diff
@@ -6,11 +6,15 @@ import UseCustomQuerySample from '../samples/UseCustomQuery';
 import { useStateValue } from '../../store/store';
 import MockAnalyticsProvider from '../../tracking/mock-analytics-provider';
 import ErrorBoundary from '../../views/ErrorBoundary';
-import { setDictionaryEndpoint, setLanguage, setSearchEndpoint } from '../../services/api/endpoints';
 import { i18n } from '../../utils';
 
 jest.mock('../../store/store');
 
+// const apiBaseEndpoint = 'http://localhost:3000/api';
```

Comment:
Any reason you're keeping this block of commented code around, or was that there before and it's the linter making this show up as new code?

Reply:
I must have been drunk. This has been removed.